.eslintrc

```diff
@@ -166,6 +166,7 @@ rules:
 
   # Babel plugin
   babel/arrow-parens: [2, as-needed]
+  babel/generator-star-spacing: 1
```

Comment:
I suggest `generator-star-spacing: [2, after]`. The only drawback is that both of these are allowed:

```js
function*()
function* ()
```

But that should be handled by another rule and should be added in a separate PR.